cms/templates/js/content-group-details.underscore

```diff
@@ -41,17 +41,18 @@
 <% if (showContentGroupUsages) { %>
   <div class="collection-references wrapper-group-configuration-usages">
     <% if (!_.isEmpty(usage)) { %>
-      <h4 class="intro group-configuration-usage-text"><%= gettext('This content group is used in:') %></h4>
+      <h4 class="intro group-configuration-usage-text"><%- gettext('This content group is used in:') %></h4>
       <ol class="usage group-configuration-usage">
         <% _.each(usage, function(unit) { %>
           <li class="usage-unit group-configuration-usage-unit">
-            <p><a href=<%= unit.url %> ><%= unit.label %></a></p>
+            <p><a href=<%= unit.url %> ><%- unit.label %></a></p>
           </li>
         <% }) %>
       </ol>
     <% } else { %>
         <p class="group-configuration-usage-text">
-        <%= outlineAnchorMessage %>
+          <!-- This contains an anchor link and therefore can't be escaped. -->
+          <%= outlineAnchorMessage %>
```

Comment:
@cahrens @efischer19: @andy-armstrong is working on code now for this, and he's working to get it into edx-platform.  Here is a related comment on [his PR](https://github.com/edx/edx-ui-toolkit/pull/30#issuecomment-196500058). 

I think the solution would be the use `HtmlUtils.HTML()` when creating `outlineAnchorMessage`, and then use `<%= HtmlUtils.escape(outlineAnchorMessage) %>` here.

Some other potential refactors include:
1. Move the HTML into this template and outside of that variable (BEST), or
2. Rename `outlineAnchorMessage` to `outlineAnchorMessageHtml` (NEXT BEST)

@andy-armstrong may want to weigh in on these recommendations.